buffother.lic

```diff
@@ -0,0 +1,47 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#buffother
+=end
+custom_require.call(%w(common common-arcana events spellmonitor drinfomon common-travel))
+class Buffother
+  include DRC
+  include DRCA
+  include DRCT
+
+  def initialize
+    arg_definitions = [
+      [
+        { name: 'recipient', regex: /\w+/, description: 'Person to buff' },
+        { name: 'waggle', regex: /\w+/, description: 'Waggle set to cast' },
+        { name: 'room', regex: /\d+/, optional: true, description: 'Room to move to' }
+      ]
+    ]
+    args = parse_args(arg_definitions)
+    @settings = get_settings
+    @recipient = args.recipient.capitalize
+    @waggle = args.waggle
+    @room = args.room
+    move_to_room
+  end
+
+  def move_to_room
+    echo "Moving to room"
```

Comment:
Consider controlling echo output via a `debug` flag. I've seen that technique used in some other scripts.

A simple example is in [dusk-labyrinth](https://github.com/rpherbig/dr-scripts/blob/master/dusk-labyrinth.lic).

The change would include adding another arg definition:
```
arg_definitions = [
  [
    ...
    { name: 'debug', regex: /debug/i, optional: true, description: 'Enable debug output' }
  ]
]
```

Then declare a debug variable that looks at UserVars first, then the script argument, then default value (usually false for off).
```
$debug_mode_buffother = UserVars.buffother_debug || args.debug || false
```

Then modify the echo statements to only output if the debug flag is set.
```
echo "Moving to room" if $debug_mode_buffother
```

Reply:
I didn't intend on committing the version with echoes included. Those will be removed. That was my own internal troubleshooting in putting the script together.